config/nginx/sites-enabled/davidrunger.com.conf

```diff
@@ -13,7 +13,7 @@ server {
     include                 nginxconfig.io/security.conf;
 
     # logging
-    access_log              /var/log/nginx/access.log combined buffer=512k flush=1m;
+    access_log              /var/log/nginx/access.log combined buffer=4k flush=1s;
```

Comment:
I think that we will flush logs whenever we meet either the 4k condition or the 1s condition. I think it will be much more common for us to reach the 1s condition first.